installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -22,6 +22,8 @@ spec:
       namespace: "kyma-installer"
     - name: "istio-kyma-patch"
       namespace: "istio-system"
+    # - name: "eventing"
```

Comment:
There are much more cr-files, shouldn't we add it to all in a commented way?

Reply:
I think `installer-cr-cluster.yaml.tpl` is enough for now. Later if we have tested the [whole functionality](https://github.com/kyma-project/kyma/issues/9230) against a local installation, we can also add it to `installer-cr.yaml.tpl`. For now we just want to have one entrypoint where we can install the event-publisher-proxy, see this [ticket](https://github.com/kyma-project/kyma/issues/9378).